manifests/globals.pp

```diff
@@ -0,0 +1,7 @@
+
+# @param $scl Enable the usage of SCL (see readme) when considering paths etc.
+#
+
```

Comment:
Whitespace around the comments breaks puppet-strings. Please also include a summary tag.
```suggestion
# @summary SUMMARY HERE
#
# @param $scl Enable the usage of SCL (see readme) when considering paths etc.
```